src/infrastructure.ts

```diff
@@ -135,4 +135,17 @@ export abstract class CodinGameInfrastructure implements Infrastructure {
       throw new Error('Unable to connect to server')
     }
   }
+
+  public getInitializationOptions (): LSPAny {
+    const files = monaco.editor
```

Comment:
might be useful to explain why we send all the files

Reply:
What about e7c21f274ad6eb2fb8951de73c676d06cfb27dbd ?